src/lib/DatePicker.svelte

```diff
@@ -29,7 +29,7 @@
   /** Update the shownDate. The date is only selected if a date is already selected */
   function updateShownDate(updater: (date: Date) => Date) {
     shownDate = updater(new Date(shownDate.getTime()))
-    if (value && shownDate.getTime() !== value.getTime()) {
+    if (!browseWithoutSelecting && value && shownDate.getTime() !== value.getTime()) {
```

Comment:
This has some unintended consequences.

`updateShownDate` is used to clamp the value within the `min` and `max` dates. So if you have `browseWithoutSelecting`, and you update the `min` date, you can now end up with a date that's outside the desired range.

It also makes it so selecting dates with the arrow keys doesn't work with `browseWithoutSelecting` (could be other things too, I haven't checked everything)